examples/object_detection_2d/object_detection_2d/seed_test_run.py

```diff
@@ -0,0 +1,154 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from argparse import ArgumentParser
+from argparse import Namespace
+from typing import Callable
+from typing import Dict
+
+import pandas as pd
+from constants import DATASET
+from constants import MODEL_METADATA
+from constants import TRANSPORTATION_LABELS
+from constants import WORKFLOW
+
+import kolena
+from kolena._experimental.object_detection import Inference
+from kolena._experimental.object_detection import Model
+from kolena._experimental.object_detection import ObjectDetectionEvaluator
+from kolena._experimental.object_detection import TestSample
+from kolena._experimental.object_detection import TestSuite
+from kolena._experimental.object_detection import ThresholdConfiguration
+from kolena._experimental.object_detection import ThresholdStrategy
+from kolena.workflow.annotation import ScoredLabeledBoundingBox
+from kolena.workflow.test_run import test
+
+MODEL_INFERENCE_PREFIX = "s3://kolena-public-datasets/coco-2014-val/results/object_detection/coco_models/"
+
+MODEL_LIST: Dict[str, str] = {
+    "yolo_r": f"YOLOR-D6 (modified CSP, {WORKFLOW})",
+    "yolo_x": f"YOLOX (modified CSP-v5, {WORKFLOW})",
+    "mask_cnn": f"Mask R-CNN (Inception-ResNet-v2, {WORKFLOW})",
+    "faster_rcnn": f"Faster R-CNN (Inception-ResNet-v2, {WORKFLOW})",
+    "yolo_v4s": f"Scaled YOLOv4 (CSP-DarkNet-53, {WORKFLOW})",
+    "yolo_v3": f"YOLOv3 (DarkNet-53, {WORKFLOW})",
+}
+
+# list of all test suites to run if a test suite is not specified
+TEST_SUITE_NAMES = [
+    f"{DATASET} :: transportation brightness [Object Detection]",
+]
+
+
+def model_alias_to_data_path(alias: str) -> str:
+    return MODEL_INFERENCE_PREFIX + alias + "/coco-2014-val_prediction.csv"
+
+
+# transforms test samples into inferences using a dataframe
+def get_stored_inferences(
+    metadata_by_image: pd.DataFrame,
+) -> Callable[[TestSample], Inference]:
+    # a function that creates an inference from a test sample
+    def infer(sample: TestSample) -> Inference:
+        try:
+            image_loc = sample.locator
+            image_name = "imgs/" + image_loc.split("/")[-1]
+            image_inferences = metadata_by_image.get_group(image_name)
+            return Inference(
+                bboxes=[
+                    ScoredLabeledBoundingBox(
+                        top_left=(float(record.min_x), float(record.min_y)),
+                        bottom_right=(float(record.max_x), float(record.max_y)),
+                        label=str(record.label),
+                        score=float(record.confidence_score),
+                    )
+                    for record in image_inferences.itertuples()
+                ],
+            )
+        except Exception as e:
+            if e is None:
```

Comment:
should this be `if e is not None`?

Reply:
wanted silent failures - had rare pandas errors before